enterprise/cli/testdata/coder_dbcrypt-rotate_--help.golden

```diff
@@ -0,0 +1,24 @@
+Usage: coder dbcrypt-rotate [flags] --postgres-url <postgres_url> --external-token-encryption-keys <new-key>,<old-keys>
+
+Rotate database encryption keys
+
+[1mOptions[0m
+      --postgres-url string, $CODER_PG_CONNECTION_URL
+          URL of a PostgreSQL database. If empty, PostgreSQL binaries will be
+          downloaded from Maven (https://repo1.maven.org/maven2) and store all
+          data in the config root. Access the built-in database with "coder
```

Comment:
Oh, is this statement still valid that `dbcrypt-rotate` will download the binary?

Reply:
Good point, that only happens when you start embedded postgres and that's not what happens here. Will update 👍